routes/items.test.js

```diff
@@ -81,6 +81,10 @@ describe("/items", () => {
       expect(itemData.items.length).toEqual(1);
       expect(itemData.items).toEqual([items[0]]);
     });
+    it("should return 404 if item does not exist", async () => {
+      const res = await request(server).get("/items/other");
+      expect(res.statusCode).toEqual(404);
+    });
```

Comment:
This is the same test as on line 27. For it to test the delete handler, it would need to make a delete request to that path.